### PR TITLE
telink: tl321x: add tl321x rf power config

### PR DESCRIPTION
**Pull request by pulin1103**
- add tl321x rf power config
- update aes config for tl3218x

### CODE REVIEW

File: tlsr9/common/b9x_rf_power.h
Comment (damien0x0023) [inline tlsr9/common/b9x_rf_power.h:1]: Please update the date from 2023 to 2023~2024.

File: tlsr9/common/b9x_rf_power.c
Comment (damien0x0023) [inline tlsr9/common/b9x_rf_power.c:1]: Please update the date at line 1 from 2023 to 2023~2024.

File: tlsr9/drivers/TL321X/aes.c
Comment (damien0x0023) [inline tlsr9/drivers/TL321X/aes.c:1]: Please confirm if we need update the date from Driver code.

File: tlsr9/drivers/TL321X/aes.h
Comment (damien0x0023) [inline tlsr9/drivers/TL321X/aes.h:1]: Same as the aes.h file.

**Comment by damien0x0023** [comment]
@andy-liu-telink @andriy-bilynskyy @mishadesh @s07641069 
It seems okay to merge it.
Please help click the merge button because I don't have that button.

**Review by damien0x0023 — COMMENTED** [review]
@andy-liu-telink @andriy-bilynskyy @mishadesh @s07641069
It seems okay to merge it.

**Comment by haiwentelink** [comment]
approve

**Review by andriy-bilynskyy — APPROVED** [review]

**Comment by andriy-bilynskyy** [comment]
Please create top level PR into Zephyr.

**Pull request merged**